### DISCUSSION TITLE
Why are there two listings for the `@canRoot` directive in the documentation?

### DISCUSSION THREAD

**Discussion by mnigh** [nuwave/lighthouse#2598] [Q&A] [answered]
Is there a reason why there are two listings for the `@canRoot` directive in the documentation, one of which does not have a prepended `@`?

- https://github.com/nuwave/lighthouse/blob/master/docs/master/api-reference/directives.md#canroot
- https://github.com/nuwave/lighthouse/blob/master/docs/master/api-reference/directives.md#canroot-1

**Comment by k0ka** [comment, marked as answer]
It's a typo. The first `canRoot` is really `@canModel`
https://github.com/nuwave/lighthouse/pull/2599
